docs/src/openapi-v1.yaml

```diff
@@ -1793,6 +1795,25 @@ components:
         trieIndex:
           type: string
           format: unsignedInteger
+    GenericExtrinsicEra:
+      type: object
+      description: |
+        The return value for eraInfo can either be `mortalEra`, or `immortalEra` and is represented as an enum in substrate. `immortalEra` meaning 
+        the transaction is valid forever. `mortalEra` consists of a tuple containing a period and phase.
+        ex: `"{"mortalEra": ["64", "11"]}"`. The Period is the period of validity from the block hash found in the signing material. 
+        The Phase is the period that this transaction's lifetime begins (and, importantly, 
+        implies which block hash is included in the signature material). If the `period` is 
+        greater than 1 << 12, then it will be a factor of the times greater than 1<<12 that 
```

Comment:
This was copied from substrate docs, i think the wording is confusing but maybe OK?

Reply:
Yea i actually totally agree, yea I just took them straight from the substrate docs. Maybe I leave out the technical part?

Reply:
yeah tbh i think it would cause more confusion than it would help

Reply:
Awesome I updated and removed the technical bit.